network/network_windows.go

```diff
@@ -337,6 +397,13 @@ func (nm *networkManager) newNetworkImplHnsV2(nwInfo *NetworkInfo, extIf *extern
 			}
 
 			log.Printf("[net] Successfully created hcn network with response: %+v", hnsResponse)
+
+			// only add net rules if it's dualStackOverlay mode and hnsNetwork is created at first time
+			if string(util.DualStackOverlay) != "" {
```

Comment:
how is this check ensuring that CNI is operating in dualstack overlay mode?

Reply:
if it's not DualStackOverlay mode (util.DualStackOverlay == "")

Reply:
so confusing as you define multiple variables with dualstack. why don't you use `epInfo.Ipv6mode`